packages/expo-application/src/Application.ts

```diff
@@ -49,6 +57,16 @@ export const applicationId: string | null = ExpoApplication
   ? ExpoApplication.applicationId || null
   : null;
 
+// @needsAudit
+/**
+ * Returns the compiled Apple `Info.plist` for the application. On Android and web, this function is `undefined`.
```

Comment:
```suggestion
 * Returns the compiled Apple `Info.plist` for the app. This function is `undefined` on Android and web.
```

We can "app" instead of "application" as we reference the former throughout our docs (for brevity).